src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt

```diff
@@ -259,4 +259,20 @@ interface VisitRepository : JpaRepository<Visit, Long>, JpaSpecificationExecutor
       " ORDER BY v.sessionSlot.slotStart,v.id",
   )
   fun findBookedVisitsForDate(prisonCode: String, date: LocalDate): List<Visit>
+
+  @Query(
+    "Select v.reference FROM visit v " +
+      " LEFT JOIN event_audit ea on ea.booking_reference = v.reference and ea.type in ('UPDATED_VISIT','BOOKED_VISIT') " +
```

Comment:
do we need UPDATED_VISIT as well? should it only be BOOKED_VISIT?

Reply:
Well it is a group by but we would always get a BOOKED_VISIT,... so I could take out UPDATED_VISIT would be faster for sure...

Reply:
Ill take it out as I was worried about  future requirement